src/sidebar/store/modules/annotations.js

```diff
@@ -441,6 +441,20 @@ const isWaitingToAnchorAnnotations = createSelector(
   annotations => annotations.some(metadata.isWaitingToAnchor)
 );
 
+/**
+ * @template S The type of state held by this store.
+ * @template A the type of actions which may be dispatched by this store.
+ * @typedef AnnotationsStore<S, A>
+ *
+ * // actions
+ * @prop {()=>A} addAnnotations
```

Comment:
@robertknight , your suggestion in slack was to just use {`typeof`}, but it feels like making this a function that returns a generic `Action` is a bit more accurate. Does that seem reasonable or do you foresee any issues with this approach?